pennylane/ops/op_math/pow_class.py

```diff
@@ -237,3 +241,14 @@ def generator(self):
         See also :func:`~.generator`
         """
         return self.z * self.base.generator()
+
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1) -> Union["Pow", Identity]:
+        if depth == 0:
+            return self
+        if self.z == 0:
+            return Identity(wires=self.wires, id=self.id)
```

Comment:
Remember that `Identity` can only accept a single wire at this point in time. 
```suggestion
            return Identity(wires=self.wires[0], id=self.id)
```

Might be a good temporary solution.